### PR TITLE
GitHub Pages Customer Testimonials

### PR DESCRIPTION
**Pull request by boardend**

### CODE REVIEW

**Comment by nirvn** [comment]
@boardend , can we close this earlier PR https://github.com/opengisch/QField/pull/2723 ?

**Comment by mbernasocchi** [comment]
@boardend I rebased on gh-pages to bring some changes I did there

File: certified-hardware copy.html [hunk 1]
```diff
@@ -0,0 +1,350 @@
+<!doctype html>
```
Comment (m-kuhn) [inline certified-hardware copy.html:1]: Is this file here on purpose?

Comment (mbernasocchi) [reply certified-hardware copy.html]: deleted

**Pull request merged**